schedule/jeos/sle/jeos-container-engines_and_tools.yaml

```diff
@@ -0,0 +1,40 @@
+---
+description: 'Test suite verifies functionality and integration of Containers (contm) and Package Hub (phub) modules'
+name: 'jeos_base+contm+phub'
+conditional_schedule:
+    bootloader:
+        MACHINE:
+            'svirt-xen-pv':
+                - installation/bootloader_svirt
+            'svirt-xen-hvm':
+                - installation/bootloader_svirt
+                - installation/bootloader_uefi
+            'svirt-hyperv-uefi':
+                - installation/bootloader_hyperv
+                - installation/bootloader_uefi
+            'svirt-hyperv':
+                - installation/bootloader_hyperv
+                - installation/bootloader_uefi
+            'uefi-virtio-vga':
+                - installation/bootloader_uefi
+            '64bit-virtio-vga':
+                - installation/bootloader_uefi
+schedule:
+    - '{{bootloader}}'
+    - jeos/firstrun
+    - jeos/record_machine_id
+    - console/system_prepare
+    - console/force_scheduled_tasks
+    - jeos/diskusage
+    - jeos/build_key
+    - console/suseconnect_scc
+    - console/consoletest_setup
+    - console/zypper_ref
+    - console/podman
+    - console/podman_image
+    - console/docker
+    - console/docker_runc
+    - console/docker_image
+    - console/docker_base_images
+    - console/zypper_docker
+    - console/docker_compose
```

Comment:
didn't you get problems with this package in JeOS? It was failing for SP2.

Reply:
Yup, a valid bug bugzilla.suse.com/show_bug.cgi?id=1167383.

docker-compose is python2 based app, for more details please refer to https://github.com/docker/compose.

In leap15.2 and TW we have replaced it with python-docker-compose, currently it is build for TW and leap15.2 only, because of other python deps issues present in building phase for sle15sp2

Reply:
shouldn't we disable this using DISTRI variable?
Same as https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/schedule/containers/extra_tests_textmode_containers.yaml#L34

Reply:
I prefer not to disable it. And besides, we have agreed that opensuse JeOS testing resides in main.pm schedule.
So if we really want to drop it, let's just remove it from the list. However, the module it is still a valid test case for SLE as well.

Reply:
if there are other packages being tested from Packagehub in JeOS, ok to leave docker-compose. We will tag the faling tests with bsc#1171913 then